python/src/internal/python-to-nix.hh

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <Python.h>
+#include <config.h>
```

Comment:
what's using config.h?

Reply:
It's included by Nix for every file: https://github.com/NixOS/nix/blob/92611e6e4c1c5c712ca7d5f9a258640662d006df/Makefile#L39

Seemingly needed for pretty much all headers, it errors when removed. I'm going to replicate the above default flag in meson though.